IBM/FHIR
# Issues #304 and #305 - remove JAXB and add java 11 to github CI

* remove remaining JAX-B references
* used matrixed build to test with both java 8 and java 11
* Update CXF to 3.3.3 for Java 11 support (CXF 3.1 is for Java 8.  Version 3.3 is the first version to support Java 11.)
* update pre-integration-test.sh to hide extraneous output
* updated labels for github action setup-java step
* explicitly call Base64.Encoder.encodeToString (previously I was using Base64.Encoder.encode which returns an int...)
* added action jobs for adoptopenjdk8 and adoptopenjdk 11
* Update to WebSocketNotificationTest
* added print env steps
* fix for integration-tests.sh
* added logging statement for date types in ParameterVisitorBatchDAO
* fix for potential liberty issue
* Update derby to 10.14.2.0 
* updated CodeGenerator and regenerated DateTime and Instant so that these will never have precision beyond 6 fractional second decimals (to ensure consistency with values parsed from strings)

Signed-off-by: Lee Surprenant <lmsurpre@us.ibm.com>
